tutor/env.py

```diff
@@ -54,7 +54,8 @@ def _prepare_environment() -> None:
             ("HOST_USER_ID", utils.get_user_id()),
             ("TUTOR_APP", __app__.replace("-", "_")),
             ("TUTOR_VERSION", __version__),
-            ("is_buildkit_enabled", utils.is_buildkit_enabled),
+            # For backwards compatability. Remove after Quince.
```

Comment:
I'm thinking we should add some sort of keyword in the comments to indicate future maintainers when to remove this. Something like `DEPR v18`. What do you think?

That keyword should also be added to tutor.images.build (see change below).

Reply:
@regisb Good thinking. Though I think `DEPR v18` would leave room for ambiguity--is it deprecated in v18, or should it be removed in v18, or removed after v18?

How about `REMOVE-AFTER-Vxx`? For example, for this template filter, we're leaving it in for Quince (v16) but it'll be gone in Redwood, so it'd be `REMOVE-AFTER-V16`.

Reply:
"REMOVE-AFTER" looks great to me!

Reply:
Great, I just pushed a new commit: https://github.com/overhangio/tutor/pull/913/commits/551c6312ad3a526770fa61906e3346b28aff818c